src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="icon" type="image/x-icon" href="./assets/images/favicon.ico" />
+		<title>To-Do List</title>
+	</head>
+	<body>
+		<section class="main">
```

Comment:
The use of the `main` tag here would be more appropriate than `section` for better accessibility.

Reply:
Thanks for suggestion.